tests/test_deformers.py

```diff
@@ -47,6 +77,70 @@ def test_raw(jam_raw):
     D = muda.deformers.TimeStretch(rate=2.0)
     six.next(D.transform(jam_raw))
 
+def freq2midi(freq):
+    return 69 + 12 * np.log2(freq/440)
+
+def midi2freq(midi):
+    return 2**((midi-69)/12)*440
```

Comment:
these functions exist in librosa (which we already depend on) .. no need to reimplement them here